### DISCUSSION TITLE
회의록 3일차

### DISCUSSION THREAD

**Discussion by wjdals83** [semi-project-team/PSMC#32] [General]
todo
피그마 찐찐찐막
업무분석 - 게시판(필요 데이터 분석)
regist->registRequest -> controller 에서 getMapping-> service(DB에 전체아이디 받기요청) -> mapper에서 전체id 조회 -> thyme에서 == 비교  -> 중복확인하기

**Comment by Absensing** [comment]
MediConnect (환자 선택) -> 게시판 (의사, 치료사 소통)

TheraphyLink (환자 선택) -> 환자 게시판 (치료사, 의사 소통)

진료 스케줄 관리 페이지 -> (조회, 등록, 수정, 삭제)
